src/HealthChecks.UI.Core/UIHealthReport.cs

```diff
@@ -9,22 +9,20 @@ namespace HealthChecks.UI.Core
      */
     public class UIHealthReport
```

Comment:
3 breaking changes here - you removed two setters and added new parameter into ctor. Is it done for proper deserialization? Will it work without these changes?

Reply:
Yes, it will works without this changes.

Reply:
Then please revert this file so your changes were backward compatible.

Reply:
It is reverted.